### PR TITLE
[Frontend] Fix: Replace the ExamplePostList to real PostList

### PR DESCRIPTION
**Pull request by badukwei**
## Summary

Replace the ExamplePostList to real PostList.

## Linked Issue

close https://github.com/social-tw/social-tw-website/issues/239

## Details

1. Replace the ExamplePostList to real PostList. So the users can read posts before login.
2. Fix the animations in welcome page.
3. Fix the styles in login page.

## Impacted Areas

packages/frontend/src/components/login/WelcomeBackgroundList.tsx
packages/frontend/src/pages/Welcome/index.tsx
packages/frontend/src/pages/Login/InternalLogin.tsx

## Verification Steps

1. 註冊並登入
2. 發布文章
3. 登出
4. 在 welcome page 可以看到發布的文章

## Checklist

-   [x] All new and existing tests pass
-   [x] Run `yarn lint:fix`

### CODE REVIEW

File: packages/frontend/src/components/login/WelcomeBackgroundList.tsx [hunk 1]
```diff
@@ -24,12 +24,15 @@ const WelcomeBackgroundList: React.FC<ScrollingModalProps> = ({
...
-            const childTop = child.offsetTop - scrollPosition + 50
-            const childBottom = childTop + child.clientHeight - 50
+            const childTop = child.offsetTop - scrollPosition
+            const childBottom = childTop + child.clientHeight
+            const middle = containerHeight / 2
 
-            const middle = (containerHeight - child.clientHeight + 200) / 2
+            const centerZoneHeight = 200
+            const centerTop = middle - centerZoneHeight / 2 + 100
+            const centerBottom = middle + centerZoneHeight / 2 - 50
```
Comment (kittybest) [inline packages/frontend/src/components/login/WelcomeBackgroundList.tsx:33]: These are not related to `child`, so could be moved outside the loop

Comment (kittybest) [reply packages/frontend/src/components/login/WelcomeBackgroundList.tsx:33]: also the `centerZoneHeight` might be changed on **mobile**?
Since now the mobile version would look like this:
image

Comment (badukwei) [reply packages/frontend/src/components/login/WelcomeBackgroundList.tsx]: 已更改

**Comment by kittybest** [comment]
實際使用情況長這樣：

https://drive.google.com/file/d/1Xb2OGinO2QasQ5WAhvt39bRE0xk12hTz/view?usp=sharing

想確定一下在設計上本來首頁這邊的文章就是可以點擊的對嗎？是的話就沒問題。

**Comment by cc03668** [comment]
@badukwei 這部分應該是不能讓 user 點進去貼文的，要再麻煩做修改～～

**Comment by badukwei** [comment]
updated!

**Review by Xiawpohr — APPROVED** [review]
Approved this pull request
